apigee_edge.module

```diff
@@ -844,6 +844,8 @@ function template_preprocess_app_credential(array &$variables) {
       }
     }
 
+    $value = (string) $value;
```

Comment:
So it was passed as on object before? Always or sometimes? If sometimes when it was passed as an object instead of a string? Shouldn't this has been fixed there instead of here?

Reply:
Sometimes it is an object, sometimes it is a string if I have seen correctly. And on the other side, sometimes it is fine to pass it as an object, sometimes it is not.

That is why I chose this solution, because it is clean and simple.

Reply:
I see, but the questions is WHY? Why sometimes it is an object? If it is an object then should not be skipped by default rather? Casting to string is not always the best solution especially if the object does not support that.
My guts says when it is an object then it is the attribute which type is AttributeProperty.  (https://github.com/apigee/apigee-client-php/blob/2.x/src/Api/Management/Entity/AppCredential.php inherited from AttributesPropertyAwareTrait).

Reply:
Couldn't you catch when exactly $value is an object?

Reply:
The error is not catchable. What happens is that the `TranslatableMarkup` object will get passed to `htmlspecialchars()` (IIRC, or a similar funciton) and then I get a warning.

Reply:
Okay, I see now. Please add a comment about this above the line and then I can accept this PR.